### PR TITLE
Leading topbar home button to welcome page

### LINKED ISSUE #6219: Replace topbar home button link from Catalog to Welcome page
Home button should lead to Welcome page.

home button

### PR DESCRIPTION
**Pull request by aleksfront**
Jumping to Welcome page instead of Catalog after home button tap.

https://user-images.githubusercontent.com/9607060/189320324-827e9379-d2fa-4ad6-b50d-09828f6f2b31.mov

Fixes https://github.com/lensapp/lens/issues/6219

Signed-off-by: alexfront <alex.andreev.email@gmail.com>

### CODE REVIEW

**Review by Nokel81 — DISMISSED** [review]

**Comment by Nokel81** [comment]
@aleksfront This shouldn't need to be updated. Those injectables will be just be corrected.

File: src/renderer/components/layout/top-bar/top-bar.tsx [hunk 1]
```diff
@@ -28,10 +26,12 @@ import closeWindowInjectable from "./close-window.injectable";
 import maximizeWindowInjectable from "./maximize-window.injectable";
 import toggleMaximizeWindowInjectable from "./toggle-maximize-window.injectable";
 import watchHistoryStateInjectable from "../../../remote-helpers/watch-history-state.injectable";
+import welcomeRouteInjectable from "../../../../common/front-end-routing/routes/welcome/welcome-route.injectable";
+import navigateToWelcomeInjectable from "../../../../common/front-end-routing/routes/welcome/navigate-to-welcome.injectable";
 
 interface Dependencies {
-  navigateToCatalog: NavigateToCatalog;
-  catalogRouteIsActive: IComputedValue<boolean>;
+  navigateToWelcomePage: NavigateToCatalog;
```
Comment (jim-docker) [inline src/renderer/components/layout/top-bar/top-bar.tsx]:
```suggestion
  navigateToWelcomePage: () => void;
```
Or create a `NavigateToWelcomePage` type?

**Review by Iku-turso — DISMISSED** [review]

File: src/renderer/components/layout/top-bar/top-bar.tsx (hunk 1 above)
Comment (aleksfront) [reply src/renderer/components/layout/top-bar/top-bar.tsx]: Good catch!

**Review by Nokel81 — APPROVED** [review]

**Review by jim-docker — APPROVED** [review]

**Comment by Nokel81** [comment]
MacOS failure is unrelated.

**Pull request merged**